### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by anna-savina**
Fixed task names and always sections in integration tests

### CODE REVIEW

File: tests/integration/targets/test_azure_manage_network_interface/tasks/main.yml [hunk 1]
```diff
@@ -85,3 +85,15 @@
         name: "{{ network_interface_virtual_network }}"
         resource_group: "{{ resource_group }}"
         state: absent
+
+    - name: Delete a Network Interface
```
Comment (nirarg) [inline tests/integration/targets/test_azure_manage_network_interface/tasks/main.yml:89]: Why did you add this? wasn't it already deleted in "Delete Network interface" task?

File: tests/integration/targets/test_azure_manage_network_interface/tasks/main.yml [hunk 2]
```diff
@@ -85,3 +85,15 @@
...
+      azure.azcollection.azure_rm_networkinterface:
+        name: "{{ network_interface_name }}"
+        resource_group: "{{ resource_group }}"
+        state: absent
+
+    - name: Delete a Network Interface's Autogenerated Security Group
```
Comment (nirarg) [inline tests/integration/targets/test_azure_manage_network_interface/tasks/main.yml:95]: This one wasn't deleted before?

File: tests/integration/targets/test_azure_manage_security_group/tasks/main.yml [hunk 3]
```diff
@@ -6,92 +6,101 @@
...
-- name: Create security group with required rules
-  ansible.builtin.include_role:
-    name: cloud.azure_ops.azure_manage_security_group
-  vars:
-    azure_manage_security_group_operation: create
-    azure_manage_security_group_resource_group: "{{ resource_group }}"
-    azure_manage_security_group_security_group:
-      name: "{{ security_group_name }}"
-      rules: "{{ security_group_rules }}"
+- name: Test role cloud.azure_ops.azure_manage_security_group
```
Comment (nirarg) [inline tests/integration/targets/test_azure_manage_security_group/tasks/main.yml:9]: I think the name before was good
This task creates the security group
Most of the tasks in this file test "cloud.azure_ops.azure_manage_security_group", its not only this task

**Review by nirarg — COMMENTED** [review]
You have many diffs as result of adding tab to all these lines
Was this change automatic, or was it intentional?

**Review by p3ck — APPROVED** [review]
LGTM

File: tests/integration/targets/test_azure_manage_network_interface/tasks/main.yml (hunk 1 above)
Comment (anna-savina) [reply tests/integration/targets/test_azure_manage_network_interface/tasks/main.yml:89]: It was added to always section, to clean up in case of failure

File: tests/integration/targets/test_azure_manage_network_interface/tasks/main.yml (hunk 2 above)
Comment (anna-savina) [reply tests/integration/targets/test_azure_manage_network_interface/tasks/main.yml:95]: It was added to always section, to clean up in case of failure

File: tests/integration/targets/test_azure_manage_security_group/tasks/main.yml (hunk 3 above)
Comment (anna-savina) [reply tests/integration/targets/test_azure_manage_security_group/tasks/main.yml:9]: I added a 'block' section which includes all previous tasks. This name for block, name of task didn't change. This structure is needed to add an 'always' section.

**Comment by anna-savina** [comment]
@nirarg In test_azure_manage_security_group I added a 'block' section which includes all previous tasks. This structure is needed to add an 'always' section. But because of this, all lines were indented and this is displayed as changes, although in fact these lines were not changed
